test/index.spec.js

```diff
@@ -0,0 +1,52 @@
+const path = require('path');
+const execa = require('execa');
+const CLI_PATH = path.resolve(__dirname, '../src/index.js');
+
+const clearComponentsDirectory = () => execa.sync('rm', ['-rf', './src/components']);
+
+describe('echo CLI test', () => {
+  it('should be able to read stdout', done => {
+    execa('echo', ['unicorns']).then(result => {
+      expect(result.stdout).toEqual('unicorns');
+      done();
+    });
+  });
+});
```

Comment:
This suite feels a bit to me like it's testing `echo` rather than anything written in userland. Just curious, in what situation would we worry about this failing? Windows, maybe?